DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md

```diff
@@ -309,11 +312,79 @@ You should see the following output in the console:
     Average throughput with API: 6370.8013 MB/s
 
     PASSED
-    ```
-    NOTE: In the performance results above, the FPGA kernel is **not** the bottleneck of the full system; the *Producer*/*Consumer* running in parallel are. The full design achieves ~87% of the maximum possible throughput (as measured by the roofline analysis).
+```
+
+## `Buffered Host-Device Streaming` Guided Discussion
+
+In this tutorial, we create a design where a **Producer** (running on the CPU) produces data into USM host allocations, a **Kernel** (running on the FPGA) processes this data and produces output into host allocations, and a **Consumer** (running on the CPU) consumes the data. Data is shared between the host and FPGA device via host pointers (pointers to USM host allocations).
+
+![](images/block_diagram.png)
```

Comment:
Same remark as in https://github.com/oneapi-src/oneAPI-samples/pull/1219 
I think that we should keep all the (FPGA) samples consistent and use the same naming convention for the folders
So to align with https://github.com/oneapi-src/oneAPI-samples/pull/1208 can you rename the `images` folder to `assets`, or disagree on the name and in that case we'll make all of these folders named `images` if you think that this is better

Reply:
> Same remark as in #1219 I think that we should keep all the (FPGA) samples consistent and use the same naming convention for the folders So to align with #1208 can you rename the `images` folder to `assets`, or disagree on the name and in that case we'll make all of these folders named `images` if you think that this is better

I updated this sample before I saw your response to change the name. I agree. Keep all samples for an area, like FPGA, as consistent as possible. Use the "assets" name for your images; I made similar changes using the "assets" folder name in later samples.

Reply:
I think that you need to push your changes then as this review still shows `images` :)